### DISCUSSION TITLE
Customize the format of List View

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [keystonejs/keystone#5348] [Feature requests]
Customizing the format of the List View

Field formatters and editors - [Built-In, Custom, Community Add-ons] .
References:
As explained in https://www.keystonejs.com/blog/roadmap-update
A number of good ideas are available from - http://tabulator.info/docs/4.0/format

- [ ] Row Height
- [ ] Column Width
- [ ] Tool Tips
- [ ] Position of Standard Buttons + Custom Actions
- [ ] Others ...

**Comment by bladey** [comment]
Thanks again for the suggestions @VinayaSathyanarayana.
